pint_xarray/tests/test_accessors.py

```diff
@@ -1543,29 +1550,30 @@ def test_reindex_like(obj, units, other, other_units, expected, expected_units,
                 dims=("x", "y"),
                 coords={"x": [10, 20, 30], "y": [60, 120]},
             ),
-            {None: "kg"},
+            "kg",
```

Comment:
is there a reason why you're unpacking the `dict`? It does not change the result in this case, but I think it is better avoid using the implicit version everywhere other than the tests for `quantify`.

```suggestion
            {None: "kg"},
```